embedded/servo_driver/servo_calibrate.cc

```diff
@@ -41,9 +41,9 @@ int keypress(int count, int key) {
   ServoDriver &servo = servos[(num - 1) / 2];
   Direction direction = num % 2 == 1 ? UP : DOWN;
   if (direction == DOWN) {
-    servo.set_percentage(std::max(0.0f, servo.get_percentage() - 1.0f));
+    servo.set_percentage(std::max(0.0, servo.get_percentage() - 0.005));
   } else {
-    servo.set_percentage(std::min(100.0f, servo.get_percentage() + 1.0f));
+    servo.set_percentage(std::min(1.0, servo.get_percentage() + 0.005));
```

Comment:
What's the +0.005?

Consider making everything `double` and removing the `_f` suffix. It's very inconsistent in the code